tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -222,16 +223,15 @@ def execute(self, ctx):
             self.topic = _choice_random_topic(ctx, prefix=self.prefix)
         if self.topic is None:
             return False
-        rpk = ctx.rpk()
         if self.total is None:
             self.current = len(
-                list(rpk.describe_topic(self.topic, tolerant=True)))
-            self.to_add = random.randint(1, 5)
-            self.total = self.current + self.to_add
+                list(ctx.rpk().describe_topic(self.topic, tolerant=True)))
+            self.total = random.randint(self.current + 1, self.current + 5)
         ctx.redpanda.logger.info(
-            f"Updating topic: {self.topic} partitions count, current: {self.current} adding: {self.to_add} partitions"
+            f"Updating topic: {self.topic} partitions count. Current partition count: {self.current} new partition count: {self.total}"
         )
-        rpk.add_topic_partitions(self.topic, self.to_add)
+        cli = KafkaCliTools(ctx.redpanda)
+        cli.create_topic_partitions(self.topic, self.total)
```

Comment:
what was the problem, is `add_topic_partitions` broken? should the PR fix it or remove?

Reply:
`add_topic_partitions` add n partitions to a topic when retried and previous operation finished with timeout it may add partitions twice to the same topic

Reply:
got it, thanks!